src/app/services/date/date.service.ts

```diff
@@ -1,10 +1,42 @@
-import { Injectable } from '@angular/core';
+import { formatNumber } from '@angular/common';
+import { inject, Injectable, LOCALE_ID } from '@angular/core';
+import { Column } from 'angular-google-charts';
 import { addDays, differenceInDays, format, isEqual } from 'date-fns';
+import { ChartData, RegistryData } from '../../models';
 
 @Injectable({
 	providedIn: 'root',
 })
 export class DateService {
+	private readonly locale = inject(LOCALE_ID);
+
+	getChartData(apiDatas: RegistryData[], start: Date, end: Date): ChartData {
```

Comment:
this method shouldn't have been migrated to `DateService`, instead we should either move back to `AppComponent` or migrate to a `ChartService`.

Reply:
updated at [65083c4](https://github.com/bitovi/opensource-analytics-dashboard/pull/11/commits/65083c476baa315e7967f73f8364a787786fabaf)